GP03FLMB/GP03FLMB_R00002_ingest.csv

```diff
@@ -1,34 +1,34 @@
-#,,GP03FLMB-RIM01-02-CTDMOH070,recovered_host,Not Expected,Data assigned to refdes after ingestion with wildcard refdes. Data not expected
-mi.dataset.driver.ctdmo_ghqr.sio.ctdmo_ghqr_ct_recovered_driver,/omc_data/whoi/OMC/GP03FLMB/R00002/cg_data/dcl00/CTDMO-H_sn11680/SBE37*11680*.hex,GP03FLMB-RIM01-02-CTDMOH070,recovered_inst,Available,
-#,,GP03FLMB-RIM01-02-CTDMOH071,recovered_host,Not Expected,Data assigned to refdes after ingestion with wildcard refdes. Data not expected
-mi.dataset.driver.ctdmo_ghqr.sio.ctdmo_ghqr_ct_recovered_driver,/omc_data/whoi/OMC/GP03FLMB/R00002/cg_data/dcl00/CTDMO-H_sn11703/SBE37*11703*.hex,GP03FLMB-RIM01-02-CTDMOH071,recovered_inst,Available,
-mi.dataset.driver.sio_eng.sio.sio_eng_sio_recovered_driver,/omc_data/whoi/OMC/GP03FLMB/R00002/cg_data/dcl00/SSIOC_SN_13282/STA*.DAT,GP03FLMB-RIS01-00-SIOENG000,recovered_host,Available,
-mi.dataset.driver.dosta_abcdjm.sio.dosta_abcdjm_sio_recovered_driver,/omc_data/whoi/OMC/GP03FLMB/R00002/cg_data/dcl00/DOSTA_sn224/DOS*.DAT,GP03FLMB-RIS01-03-DOSTAD000,recovered_host,Available,
-mi.dataset.driver.phsen_abcdef.phsen_abcdef_recovered_driver,/omc_data/whoi/OMC/GP03FLMB/R00002/cg_data/dcl00/PHSEN_snP0102/SAMI*.txt,GP03FLMB-RIS01-04-PHSENF000,recovered_inst,Available,
-mi.dataset.driver.phsen_abcdef.imodem.phsen_abcdef_imodem_recovered_driver,/omc_data/whoi/OMC/GP03FLMB/R00002/cg_data/dcl00/PHSEN_snP0102/PHS*.DAT,GP03FLMB-RIS01-04-PHSENF000,recovered_host,Available,
-mi.dataset.driver.flort_dj.sio.flort_dj_sio_recovered_driver,/omc_data/whoi/OMC/GP03FLMB/R00002/cg_data/dcl00/FLORD_sn1124/FLO*.DAT,GP03FLMB-RIS01-05-FLORTD000,recovered_host,Available,
+parser,filename_mask,reference_designator,data_source,status,notes
+mi.dataset.driver.sio_eng.sio.sio_eng_sio_recovered_driver,/omc_data/whoi/OMC/GP03FLMB/R00002/cg_data/dcl00/MSIOC_SN_10118/STA*.DAT,GP03FLMB-RIM01-00-SIOENG000,recovered_host,Available,
+mi.dataset.driver.adcps_jln.sio.adcps_jln_sio_telemetered_driver,/omc_data/whoi/OMC/GP03FLMB/R00002/cg_data/dcl00/ADCP_sn20504/ADC*.DAT,GP03FLMB-RIM01-02-ADCPSL007,recovered_host,Available,
+mi.dataset.driver.adcps_jln.adcps_jln_driver,/omc_data/whoi/OMC/GP03FLMB/R00002/cg_data/dcl00/ADCP_sn20504/*.000,GP03FLMB-RIM01-02-ADCPSL007,recovered_inst,Available,
+mi.dataset.driver.ctdmo_ghqr.sio.ctdmo_ghqr_sio_ct_recovered_driver,/omc_data/whoi/OMC/GP03FLMB/R00002/cg_data/dcl00/CTDMOs_all/CTD*.DAT,GP03FLMB-RIM01-02-CTDMOG000,recovered_host,Available,Wildcard refdes. 
+#,/omc_data/whoi/OMC/GP03FLMB/R00002/cg_data/dcl00/CTDMOs_all/CTD*.DAT,GP03FLMB-RIM01-02-CTDMOG060,recovered_host,Available,Data assigned to refdes after ingestion with wildcard refdes. 
+#mi.dataset.driver.ctdmo_ghqr.sio.ctdmo_ghqr_ct_recovered_driver,/omc_data/whoi/OMC/GP03FLMB/R00002/cg_data/dcl00/CTDMO-G_sn11646/SBE37*11646*.hex,GP03FLMB-RIM01-02-CTDMOG060,recovered_inst,Available,

```

Comment:
I think this line needs to be uncommented.

Reply:
Agreed